Makefile

```diff
@@ -72,7 +72,7 @@ unittests:
 	@echo "Running unit tests"
 	@tox -e unit -- ${UNIT_ARGS}
 
-functional: 
+functional:
```

Comment:
Shouldn't we trigger build when running functional tests?

Reply:
No, with `pytest-operator` we are doing [build](https://github.com/canonical/charm-simple-streams/blob/main/tests/functional/test_charm.py#L13) in func test.